lib/rubocop/cop/style/mixin_usage.rb

```diff
@@ -48,24 +48,22 @@ class MixinUsage < Cop
 
         def_node_matcher :include_statement, <<-PATTERN
           (send nil? ${:include :extend :prepend}
-            (const nil? _))
+            const)
         PATTERN
 
         def on_send(node)
           return unless (statement = include_statement(node))
-          return unless top_level_node?(node)
+          return unless implicit_style?(node)
 
           add_offense(node, message: format(MSG, statement: statement))
         end
 
         private
 
-        def top_level_node?(node)
-          if node.parent.parent.nil?
-            node.sibling_index.zero?
-          else
-            top_level_node?(node.parent)
-          end
+        def implicit_style?(node)
+          return true if node.parent.nil?
```

Comment:
I think:

```ruby
def implicit_style?
  node.parent || !node.macro?
end
```

is slightly easier to understand. WDYT?

Reply:
Right! I changed the method name to `accepted_include?`, so I reversed the condition, but I think it is easy to understand 😃